test/apps/iodemo/io_demo.cc

```diff
@@ -217,6 +227,54 @@ class P2pDemoCommon : public UcxContext {
         return conn->send_io_message(m->buffer(), opts().iomsg_size, m);
     }
 
+    bool send_recv_data_as_chunks(UcxConnection* conn, size_t data_size, uint32_t sn,
+                send_recv_data_t send_recv_data, UcxCallback* callback = EmptyCallback::get()) {
+        size_t left_size = data_size;
+        while (left_size > 0) {
+            size_t sz = (left_size >= opts().chunk_size) ? opts().chunk_size : left_size;
+            if (send_recv_data == SEND_DATA) {
+                conn->send_data(buffer(data_size - left_size), sz, sn, callback);
+            } else {
+                conn->recv_data(buffer(data_size - left_size), sz, sn, callback);
+            }
+            left_size -= sz;
+        }
+        return true;
```

Comment:
why need to return 'bool' and not 'void'? does the function ever return false?

Reply:
To align with conn->send_data/recv_data. Already added more return values.

Reply:
this is done only in ucx_wrapper, no need to have it in the demo client/server

Reply:
@yosefe Fixed. Thanks!